inst/skeletons/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to this report factory\!
+
+The factory will help you handle multiple `rmarkdown` reports at the
+same time. Put your `.Rmd` files in `report_sources`, refer to any
+external files in the `.Rmd` using `here::here()`, and you’re sorted.
+
+## How the factory is organised
+
+  - `report_sources`: (mandatory) put your `.Rmd` documents there
+    (subfolders are OK)
+
+  - `data/`: (recommended) put your data in this folder (subfolders are
+    OK)
+    
+      - `data/raw`: to store raw data, as read-only
```

Comment:
I think this file needs updating; check that the default structure outlined is correct, and update with new functions.

Reply:
to be checked but hopefully closed with https://github.com/reconhub/reportfactory/pull/68/commits/c022fc73b1360e4806282a532c3079af6912575f